utils.py

```diff
@@ -405,13 +412,23 @@ def get_time_in_millisecs(datetime_obj):
 
 
 def get_current_time_in_millisecs():
-    """Returns time in milliseconds since the Epoch."""
+    """Returns time in milliseconds since the Epoch.
+
+    Returns:
+       float. Returns time in millisecond.
```

Comment:
The description should be about the return and not about the function, so remove the `Returns` keyword from the description. (Ditto elsewhere.)

Reply:
done